src/app/shareUrl/sharelink-url.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright 2022 The Triple Banana Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+export class ShareLink {
+  static readonly SHARE_LINK_PARAM = "?sharelink="
```

Comment:
I'm curious about the `shareLink` vs `sharelink`.

Reply:
in the url, I thought camel case is not suitable on the other hands,  in code camel case used

Reply:
Ack